diff --git a/codex-rs/Cargo.lock b/codex-rs/Cargo.lock
index 127a7037c2..a52909d4f0 100644
--- a/codex-rs/Cargo.lock
+++ b/codex-rs/Cargo.lock
@@ -4135,6 +4135,7 @@ dependencies = [
  "clap",
  "codex-ansi-escape",
  "codex-app-server-client",
+ "codex-app-server-daemon",
  "codex-app-server-protocol",
  "codex-arg0",
  "codex-cli",
diff --git a/codex-rs/cli/src/main.rs b/codex-rs/cli/src/main.rs
index 3ae8bf7d60..c336bf0726 100644
--- a/codex-rs/cli/src/main.rs
+++ b/codex-rs/cli/src/main.rs
@@ -2289,12 +2289,14 @@ async fn run_interactive_tui(
         }
         Err(err) => return Err(err),
     };
+    let default_local_daemon = false;
     let start_tui = || {
         codex_tui::run_main(
             interactive.clone(),
             arg0_paths.clone(),
             codex_config::LoaderOverrides::default(),
             remote_endpoint.clone(),
+            default_local_daemon,
         )
     };
     let mut attempted_backups = HashSet::new();
